src/en/proposals/pda-messaging.md

```diff
@@ -0,0 +1,71 @@
+## PDA messaging program and server
+
+[Julian, VasilisThePikachu, Unapproved]
+
+*(Taken by [Julian doc](https://hackmd.io/iu2yK9bcQb-veuCOLl-FYw?both#Optional-Channels-and-Department-based-Channels) in hackmd and modified a lil. Mostly replacing "email" to "message", "Email address" to "user/user id" and adding some of my own twists. Julian was fine with this if i understood correctly (i was in vc with em))*
+
+*(This is mostly taken from how PDA messages work in ss13)*
+Allows sending messages to others using PDAs
+
+### Message storage
+Messages are stored on a server most likely will be stored in telecoms. There can be one server per station, others on the same station won't be used unless the first one loses power or gets destroyed.
+
+*Optional* The active server synchronises itself with all of the inactive servers on the same station (This happens inside the system directly, no device networking here).
+
+### One active / Multiple inactive server model
+
+(This talks about some refactor stuff and Julian told me they forgot to paste the link, im keeping it to be safe in case its actually useful.)
+
+The one active / multiple inactive server model uses the system that will get refactored into its own system from the crew monitoring server [link text]() 
+
+The messaging client system will use the `GetActiveServer` method of the message server system to retrieve the active server if the client doesn't have a server set yet or that server timed out. *This is also from the system that gets refactored out.*
+
+### Sending and receiving messages
+
+When sending a message to someone via the program the PDA sends the message together with an 'user id' to the server and the server will send the message to the target device. 
```

Comment:
Who would maintain the player character to UID mapping? e.g. a new player arrives on the station, does their ID get added to the list automatically? What happens if they die or move off station? How does the HoP change this mapping if somebody receives a new ID?

Reply:
The chat server could maintain the id's.

Late joiners will get a uid assigned to them when they enter the station.

If they die, that's not a problem. The place the uid is stored is the ID. As long as that ID is inserted in a PDA it will keep receiving messages.
If the ID card is off station then potentially queuing the messages or doing a "message failed to send" sounds good too.

Hop could use the id modification computer for this, although this could be rd's job too other then maintaining the server.

I will add these points and your other review in the doc soon